content/departments/engineering/dev/tools/scaletesting.md

```diff
@@ -0,0 +1,79 @@
+# ScaleTesting Instance
+
+This page documents resources that are relevant for engineers looking to run Sourcegraph at scale as a preemptive measure to uncover bugs that
+may only be caught when operating under particular scale. Typically, this enables to approximate a customer instance usage pattern to ensure
+that our application will perform within acceptable boundaries in that context.
+
+Join the [#wg-test-at-scale](https://sourcegraph.slack.com/archives/C040LV3PS4C) Slack channel to join conversations about its use, both from theperspective of an engineer using it to test things or to follow or contribute to its development.
+
+## Scaling VS Customer Tiers
+
+It is important to make the distinction that by scale here, we're referring to the scale of operations from the perspective of a customer. So we're talking in terms of number of repositories, their size and the number of users. This is not directly about observing far we can scale horizontally or vertically a given service, though this can be achieved though the same mean.
+
+The common vocabulary being used to talk about scaling from the persective of a customer is described in details over [tiers of strategic accounts](https://docs.google.com/spreadsheets/d/1n-KfGc8m1w09rIzNKm5tRxAYmP4-w11CVOCplMvVazk/edit#gid=1172385107). Therefore, we it's best to use it (_LARGE_, _XL_, _2XL_ and _3XL_) to frame which kind of customer a test would target.
+
+## ScaleTesting Instance
+
+This instance is entirely dedicated to peform manual testing at this stage and should not be used for other purpose. It is assumed that all the data associated with that instance can be discarded at the discretion of engineers performing tests on it or by the Dev Experience team.
+
+:right_arrow: If you plan to run a test, announce yourself on [#wg-test-at-scale](https://sourcegraph.slack.com/archives/C040LV3PS4C) to ensure you're the only one using it at the moment.
+
+It is deployed in its own Google Cloud Project and is maintained by the Developer Experience team. This is very much a collaborative effort and any help to improve it is welcomed.
+
+> :bulb: There are zero alerting enabled on that instance, you are on your own and are expected to reach out for help if you notice something erratical with the deployment itself.
+
+## TL;DR to conduct a test
+
+> :warning: Do not ever populate that instance with customer data. **Only use open source or synthetic data.**
+
+1. Have access to the Google Project: TODO
+2. Be familiar with our Observability stack.
+3. Be familiar with our Infrastructure code.
+4. Join [#wg-test-at-scale](https://sourcegraph.slack.com/archives/C040LV3PS4C) and announce yourself.
```

Comment:
Idea: `sg scaletest announce code-change/testing` which posts a standard message in #wg-test-at-scale, we could also have crude mechanism like a file in gdrive storing the "lock" ?

Reply:
Yes, those are low hanging fruits we can do once we have shipped the first iteration!